web/profiles/joinup/config/install/core.entity_form_display.rdf_entity.collection.propose.yml

```diff
@@ -22,11 +22,83 @@ dependencies:
     - image.style.thumbnail
     - rdf_entity.rdfentity.collection
   module:
+    - field_group
     - image
     - inline_entity_form
     - link
     - rdf_entity
     - text
+third_party_settings:
+  field_group:
+    group_general:
+      children:
+        - group_main
+        - group_description
```

Comment:
Yes, that was the idea. The frontend team will restyle the tabs and move the description underneath the tab title.